content/client-lib-development-guide/features.textile

```diff
@@ -299,7 +299,13 @@ h3(#realtime-connection). Connection
 ** @(RTN15b)@ In order for a connection to be resumed and connection state to be recovered, the client library reconnects to the websocket endpoint with two additional querystring params:
 *** @(RTN15b1)@ @resume@ is the private connection key assigned to the connection when the first @CONNECTED@ @ProtocolMessage@ was received
 *** @(RTN15b2)@ @connection_serial@ is the most recent @ProtocolMessage#connectionSerial@ received from Ably or @Connection#serial@ which should be identical
-** @(RTN15c)@ When a new connection is made using the @resume@ and @connection_serial@ params, the client library must consider the resume to have failed if the @CONNECTED@ @ProtocolMessage@ contains an error or if the @Connection#id@ has changed. When a @Connection@ resume has failed, all channels should be detach with a suitable error reason
+** @(RTN15c)@ The system's response to a resume request will be one of the following:
+**** @(RTN15c1)@ @CONNECTED@ @ProtocolMessaeg@ with the same @connectionId@ as the current client, and no error. In this case, the server is indicating that the resume succeeded, all channels are still attached, and all backlog messages are available
+**** @(RTN15c2)@ @CONNECTED@ @ProtocolMessaeg@ with the same @connectionId@ as the current client, and an error in @errorInfo@. In this case, the server is indicating that the resume succeeded but with a non-fatal error, all channels are still attached, and some backlog messages may be unavailable. The @ErrorInfo@ received should be emitted on the client @Connection@ as well as all @ATTACHED@ @Channel@ objects. The exact use case for this partial success state is still being reviewed, however this does not effect the client library behaviour
```

Comment:
Thanks re: typo.

Re: `errorInfo`, that is worrying.  Looking at the spec TR4h at http://docs.ably.io/client-lib-development-guide/features/, that is certainly not the case.  I will update there as well.